projects/packages/forms/src/blocks/field-rating/style.scss

```diff
@@ -0,0 +1,53 @@
+/* Rating field block – front-end styles */
+@import url(https://fonts.googleapis.com/css2?family=IBM+Plex+Sans+JP&display=swap&text=♥★);
```

Comment:
I'm wondering a bit about this approach to loading our stars. It seems like an unusual approach, and requires a network request. I think there are also GDPR considerations ([Do google fonts violate GDPR?](
https://www.cookieyes.com/documentation/google-fonts-and-gdpr/ )) Not sure what the best alternative is - possibly using svgs? But that may change the architecture of the block markup and behavior.